docs/en_us/platform_api/source/courses/courses.rst

```diff
@@ -0,0 +1,174 @@
+.. _Courses API Courses Resource:
+
+########################################
+Courses API Courses Resource
```

Comment:
@srpearce @bderusha Perhaps a solution is to list the task/method/endpoint in the catalogs overview file, but make the endpoint text link to this topic? That way the doc for each resource is kept in that resource's file (in this case course/courses.rst) (beginning to sound like "curses") but because the task is a catalogs task it can be listed in that file?